frontend/src/metabase/admin/embedding/components/ThemeListing/EmbeddingThemeCardPreview.tsx

```diff
@@ -0,0 +1,6 @@
+import { Box } from "metabase/ui";
+
+// TODO(EMB-942): add embedding theme preview placeholder card
+export const EmbeddingThemeCardPreview = () => (
+  <Box h="200" bg="bg-secondary" w="100%" />
```

Comment:
This is a placeholder for the super fancy dynamic placeholder card. I'll implement this separately because it's quite fancy:

<img width="200" height="200" alt="Image" src="https://github.com/user-attachments/assets/47637b37-39dd-4812-815d-68d27cdc40d3" />